Modules/mathmodule.c

```diff
@@ -2163,6 +2163,27 @@ math_ldexp_impl(PyObject *module, double x, PyObject *i)
     } else {
         errno = 0;
         r = ldexp(x, (int)exp);
+#if _MSC_VER
```

Comment:
Hmm, I'd always assumed this should be `#ifdef`, but I guess if the build is passing on other platforms then it doesn't have to be.

Reply:
Then it will hurt performance for all platforms.

Reply:
I didn't realise all platforms defined the MSVC-specific compiler constant! No doubt we have other sections of code that are also being compiled on non-Windows platforms, if this is the case.